build/run.js

```diff
@@ -155,32 +147,40 @@ commands.build.rust = async function (argv) {
         '--target',
         'web',
         '--out-dir',
-        paths.dist.wasm.root,
+        paths.wasm.root,
         '--out-name',
         'ide',
         crate,
     ]
+    // Release builds are disabled on M1 Mac because of https://github.com/rustwasm/wasm-pack/issues/913
```

Comment:
The linked issue is closed and apparently release was made around then. Is it possible to remove this workaround now?

Reply:
Yeah, I'm sorry, I removed it after you started a review.